### ISSUE TITLE
Menu: focus indicator is not going the entire width

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#52911]
Refs: #52854

It looks like the focus indicator is only visible half width:

![image](https://user-images.githubusercontent.com/900690/41905053-524492a2-793a-11e8-8765-401527b8e620.png)

**Comment by beastdestroyer** [comment]
The focus indicator is on  by default, however shouldn't it be off, since most people will be using mice not the arrow keys, and then it will automatically appear once you use the keys? Also it isn't half-width on the "Tasks", "Help" menus, nor the gear in the bottom left.

**Comment by octref** [comment]
Agreed - the focus indicator should include the whole label, and accessibility wise it should read over the Label + Shortcut.

**Comment by bpasero** [comment]
@sbatten I am no longer seeing the focus indicator at all now, did something change?

![image](https://user-images.githubusercontent.com/900690/42039430-917dc510-7aed-11e8-8dca-bb3cb89f92bf.png)

**Comment by sbatten** [comment]
@bpasero it seems the menu.css and shell.css get loaded in the opposite order in OSS which I don't know why, but that might be why you are not seeing it. It still repros in insiders. While I was figuring that out, I discovered that PR #53289 's fix actually addresses this issue as well as makes the screen reader read the keyboard shortcuts and mnemonics correctly. 🎉

**Comment by bpasero** [comment]
I commented.
